notebooks/20220217_summarize_rnaseq_sourmash_gather_results.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<p>Is SRX249887 contaminated? Or mislabeled? Is there a reason that none of its reads map to P. aeruginosa?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/2022-microberna/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>